src/status_im/ui/components/tabbar/styles.cljs

```diff
@@ -115,11 +108,21 @@
    :right            0
    :left             0})
 
-(defn tabs-wrapper [keyboard minimized inset]
-  (merge {:padding-bottom   inset
-          :elevation        8
-          :padding-top      (if minimized 0 tabs-diff)
-          :background-color :white}
+(defn tabs-wrapper [keyboard visible]
+  (merge {:padding-horizontal 8
+          :elevation          8
+          :left               0
+          :right              0
+          :bottom             0
+          :transform          [{:translateY
+                                (animation/interpolate visible
+                                                       {:inputRange  [0 1]
+                                                        :outputRange [0 tabs-height]})}]}
          (when keyboard
-           {:position :absolute
-            :bottom   (- tabs-height)})))
+           {:position :absolute})))
+
+(defn space-handler [visible inset]
+  {:height     (+ inset minimized-tabs-height)
+   :margin-top (animation/interpolate visible
```

Comment:
Because of react-native animation does not support layout changes (margin-top), this animation value should be on js thread. Keeping 2 animation values allows us to animate this invisible block and make space for the native animation which uses translate.

This also solves the problem that touchable inside translate move do not receive touches https://github.com/facebook/react-native/pull/26374

Reply:
could you add this comment in the code ? would be useful